### PR TITLE
Added Printer Health Check Component!

### PR DESCRIPTION
**Pull request by Annabel1054**

### CODE REVIEW

File: src/Pages/2DPrinting/2DPrinting.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
-import React, { useState } from 'react';
+
```
Comment (jerrylee17) [inline src/Pages/2DPrinting/2DPrinting.js:1]: Remove this line

File: src/Pages/2DPrinting/2DPrinting.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
+import React, { useState, useEffect } from 'react';
```
Comment (jerrylee17) [inline src/Pages/2DPrinting/2DPrinting.js:2]: useEffect not used. Remove it

File: src/Pages/2DPrinting/2DPrintingHealthCheck.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  Spinner,
+  Container
+} from 'reactstrap';
+
+export default function PrintingHealthCheck(props) {
+  const [loading, setLoading] = useState(true);
+  const [printerHealthy, setPrinterHealthy] = useState(false);
+
+  useEffect(() => {
+    async function checkPrinterHealth() {
```
Comment (jerrylee17) [inline src/Pages/2DPrinting/2DPrintingHealthCheck.js:13]: This function should be declared and implemented outside of useEffect. Removes the clutter in useEffect function

File: src/Pages/2DPrinting/2DPrinting.js [hunk 4]
```diff
@@ -1,10 +1,11 @@
...
 import './2D-printing.css';
 import ConfirmationModal from
   '../../Components/DecisionModal/ConfirmationModal.js';
 import Header from
   '../../Components/Header/Header.js';
-import { registerPlugin } from 'react-filepond';
+import { FilePond, registerPlugin } from 'react-filepond';
```
Comment (jerrylee17) [inline src/Pages/2DPrinting/2DPrinting.js:8]: FilePond not used

**Review by jerrylee17 — APPROVED** [review]
Works

**Review by IsitaB — APPROVED** [review]
works for me

**Pull request merged**